src/main/java/org/gatorgradle/config/GatorGradleConfig.java

```diff
@@ -194,6 +178,10 @@ public void parseHeader() {
       fastBreakBuild = file.getHeader("fastfail").asBoolean();
     }
 
+    if (file.hasHeader("idcommand")) {
```

Comment:
I'd like to also include the secondary option `idcmd`; this could be done similarly to `revision` and `version` as below.